CHANGELOG.md

```diff
@@ -26,7 +27,7 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 ### Clean Up
 - *Add item here*
 ### Notes For Existing Projects
-- *Add item here*
+- Ensure to run the SQL patch to create and update the `AcquisitionDate` field in the `files` table (PR #6892)
```

Comment:
I don't think this should be here.. it'll be part of the release patch when it's compiled.

Reply:
i agree. unless projects need to do a manual step other than the regular patch

Reply:
ok

Reply:
removed in the last commit